src/BatchResult.php

```diff
@@ -46,26 +27,16 @@ public function getResponses()
      *
      * @return ResponseInterface
      *
-     * @throws UnexpectedValueException
+     * @throws \UnexpectedValueException If request is not found
```

Comment:
this is only for successful responses, right? so we should say

If request was not part of the batch request or was not successful.